### DISCUSSION TITLE
Globe light position

### DISCUSSION THREAD

**Discussion by slaviczavik** [maplibre/maplibre-gl-js#5224] [General]
Hello,

I know the Globe is not stable yet, but I would like to ask if the light is working correctly for the globe projection.

I have create [simple example](https://jsbin.com/yiredilowi/edit?html,output) with position set to `[1.15, 0, 0]` an "sun" is shinning upon Pacific ocean, but I would say it should be shinning upon Europe?

Thanks for advice!

**Comment by HarelM** [comment]
There was an effort to support light for the atmosphere, not sure how well it works, I would try with something more basic before trying to understand how it looks on satellite imagery...

**Reply by slaviczavik** [reply]
I have updated an example and created [issue](Closing this discussion – [issue](https://github.com/maplibre/maplibre-gl-js/issues/5229) has been created.) for it.

**Comment by slaviczavik** [comment]
Closing this discussion – [issue](https://github.com/maplibre/maplibre-gl-js/issues/5229) has been created.
